### ISSUE TITLE
Investigate raster image to vector conversion

### ISSUE THREAD

**Issue by surma** [GoogleChromeLabs/squoosh#986]
- [Potrace](http://potrace.sourceforge.net/) (what Inkscape uses)
- [ImageTracerJS](https://github.com/jankovicsandras/imagetracerjs)
- [AutoTrace](https://github.com/autotrace/autotrace)

**Comment by jakearchibald** [comment]
We'd need to make it clear that this is a bitmap trace rather than "SVG output". Otherwise, if you load an SVG and select SVG output, it might not be clear that it's doing a round trip through a bitmap.

**Comment by surma** [comment]
Yeah. Could also just hide this when the input is a vector image?

**Comment by jakearchibald** [comment]
Maybe. Although maybe there are times what you might want to do the round trip.

SVG with lots of gradients -> Bitmap -> Reduce palette -> Trace to SVG

**Comment by surma** [comment]
Oh yeah, good point.

**Comment by kosamari** [comment]
Note: listing codec & UI label because this will be a shared feature  (I know it is not exactly a codec :)
